config/opal_config_asm.m4

```diff
@@ -145,9 +146,8 @@ static void test1(void)
     ompi128 ptr      = { .fake = { 0xFFEEDDCCBBAA0099, 0x8877665544332211 }};
     ompi128 expected = { .fake = { 0x11EEDDCCBBAA0099, 0x88776655443322FF }};
     ompi128 desired  = { .fake = { 0x1122DDCCBBAA0099, 0x887766554433EEFF }};
-    bool r = atomic_compare_exchange_strong (&ptr.real, &expected.real,
-                                             desired.real, true,
-					     atomic_relaxed, atomic_relaxed);
+    bool r = atomic_compare_exchange_strong (&ptr.real, &expected.real2,
```

Comment:
@hjelmn I don't understand this function call - there is no `real` field any more in `ompi128`. You renamed it to `real2`. Should all these fields be renamed?

Reply:
Ah, I'm sorry - I misread your edit on line 138. My bad.